app/assets/javascripts/chf_image_viewer.js

```diff
@@ -438,13 +438,6 @@ jQuery(document).ready(function($) {
       event.preventDefault();
       var id = this.getAttribute('data-member-id');
       chf_image_viewer().show(id);
-
-      // GA
-      _gaq.push(['_trackEvent',
-           'ImageViewer',
-           'fileSetId',
-           id
-          ]);
     });
```

Comment:
Yep, tested and working as long as we are okay with the requirement for a class instead of being able to use just data attributes.